src/android/app/src/main/jni/native.cpp

```diff
@@ -795,4 +799,29 @@ void Java_org_citra_citra_1emu_NativeLibrary_disableTemporaryFrameLimit(JNIEnv*
     Settings::is_temporary_frame_limit = false;
 }
 
+void Java_org_citra_citra_1emu_NativeLibrary_playTimeManagerInit(JNIEnv* env, jobject obj) {
+    play_time_manager = std::make_unique<PlayTime::PlayTimeManager>();
+}
+
+void Java_org_citra_citra_1emu_NativeLibrary_playTimeManagerStart(JNIEnv* env, jobject obj,
+                                                                  jlong title_id) {
+    ptm_current_title_id = title_id;
+    play_time_manager->SetProgramId(static_cast<u64>(title_id));
```

Comment:
Should we add null guards?

Reply:
I'm pretty sure that after the changes in my most recent commit it's impossible for `title_id` to be null, so this should be fine.

Reply:
I was talking about `play_time_manager`